README.md

```diff
@@ -184,12 +184,11 @@ USER root
 
 RUN apk add --update --virtual .build-deps \
         sudo build-base ruby-dev \
-
- # cutomize following instruction as you wish
- && sudo -u fluent gem install \ 
+ `# customize following instruction as you wish` \
```

Comment:
This might be an alternative:
```
# elasticsearch record-reformer provided as examples below
# customize plugins as you wish
RUN apk add --update --virtual .build-deps \
        sudo build-base ruby-dev \
 && sudo -u fluent gem install \
        fluent-plugin-elasticsearch \
        fluent-plugin-record-reformer \
 && sudo -u fluent gem sources --clear-all \
 && apk del .build-deps \
 && rm -rf /var/cache/apk/* \
           /home/fluent/.gem/ruby/2.3.0/cache/*.gem
```